FhirToDataLake/azurepipelines/OverallTest/DeployContainerInstance.json

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
Is this template used in overall test?

Reply:
No, the overall test uses Container Instance that been deployed be this template, I just want to add this for reference.

Reply:
Got it. Thanks!